qemu/tests/virtio_fs_share_data.py

```diff
@@ -760,6 +794,12 @@ def check_security_label(file, folder, xattr_name):
                     session = vm.reboot(session)
                 if se_mode_guest_before.lower() == "permissive":
                     session.cmd("setenforce Permissive")
+        if os_type == "windows" and len(params.get("filesystems").split()) == 1:
+            error_context.context("Unmount fs with WinFsp.Launcher.z")
+            winfsp_serv_stop_cmd = params["winfsp_serv_stop_cmd"]
+            for fs in params.objects("filesystems"):
+                fs_target = fs_params.get("fs_target")
+                session.cmd(winfsp_serv_stop_cmd % fs_target)
```

Comment:
I didn't understand this part since the variant `fs` is not called in the following `for` statement.
In the other words, what's the purpose of the `for` statement?
Would you like to explain it?
Thanks.

Reply:
Yes, I missed it, it should be used to get fs_params.
I already update, pls have a check, thanks.